Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -1555,7 +1555,22 @@ def panorama_create_address_command(args: dict):
     """
     address_name = args['name']
     description = args.get('description')
-    tags = argToList(args['tag']) if 'tag' in args else None
+    if tags := argToList(args.get('tag', [])):
+        xpath = "/config/devices/entry[@name='localhost.localdomain']"
+        if DEVICE_GROUP:
+            xpath += f"/device-group/entry[@name='{DEVICE_GROUP}']/tag"
+        if VSYS:
+            xpath += "/vsys/entry[@name='vsys1']/tag"
+        result = http_request(URL, 'GET', params={'type': 'config', 'action': 'get', 'key': API_KEY, 'xpath': xpath})
+        entries = result.get('response', {}).get('result', {}).get('tag', {}).get('entry', [])
+        if isinstance(entries, dict):
+            entries = [entries]
+        exist_tags = [entry.get('@name') for entry in entries]
+        for tag in tags:
+            if tag not in exist_tags:
+                raise DemistoException(
+                    f'Failed to create the address object since the tag `{tag}` does not exist.'
+                )
```

Comment:
```suggestion
        existing_tags = set([entry.get('@name') for entry in entries])
        if non_existent_tags := existing_tags - tags:
            raise DemistoException(
                    f'Failed to create the address object since the following tags `{non_existent_tags}`do not exist.'
                )
```

maybe work with sets? would be easier in my opinion.